packages/eslint-config-core-ts/index.js

```diff
@@ -9,16 +9,13 @@ module.exports = {
     node: true,
   },
   extends: [
-    'eslint:recommended',
+    '@infinumjs/eslint-config-core-js',
     'plugin:@typescript-eslint/eslint-recommended',
     'plugin:@typescript-eslint/recommended',
-    'prettier',
     'prettier/@typescript-eslint',
-    'prettier/standard'
   ],
   plugins: [
     '@typescript-eslint',
```

Comment:
Did you check that the plugins are appended and not replaced?

Reply:
Good point, will check.